app/helpers/ama_layout_path_helper.rb

```diff
@@ -64,6 +64,6 @@ def membership_esso_reload_path
   end
 
   def driveredonline_path
-    "#{Rails.configuration.driveredonline_site}/login"
+    "#{Rails.configuration.driveredonline_site}"
```

Comment:
Can drop the `"#{ }"` parts to just return that value directly, since there's no more plain text.